src/main/java/com/jmb/springfactory/controller/GenericController.java

```diff
@@ -0,0 +1,48 @@
+package com.jmb.springfactory.controller;
+
+import java.io.Serializable;
+import java.util.List;
+
+import com.jmb.springfactory.exceptions.NotFoundException;
+import com.jmb.springfactory.exceptions.ServiceLayerException;
+import com.jmb.springfactory.model.dto.BaseDto;
+
+public interface GenericController<D extends BaseDto, ID extends Serializable> {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this generic name to match the regular expression '^[A-Z][0-9]?$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00119)